homeassistant/components/husqvarna_automower/lawn_mower.py

```diff
@@ -106,3 +126,21 @@ async def async_dock(self) -> None:
             raise HomeAssistantError(
                 f"Command couldn't be sent to the command queue: {exception}"
             ) from exception
+
+    async def async_start_for(self, duration: int) -> None:
+        """Let the mower mow for a given time."""
+        try:
+            await self.coordinator.api.commands.start_for(self.mower_id, duration)
+        except ApiException as exception:
+            raise HomeAssistantError(
+                f"Command couldn't be sent to the command queue: {exception}"
```

Comment:
I'd simplify this message for all the service calls. We don't need to explain the implementation details of the library in the error message.
```suggestion
                f"Failed to send command: {exception}"
```

Reply:
Done in https://github.com/home-assistant/core/pull/117269/commits/fb6227bd55938efb911a008811ee2b820e4db6c4 , I also used a decorator, so we don't have to much duplicates. I can use the decorator in another PR for the other platforms.